test/snowflex_test.exs

```diff
@@ -575,85 +540,85 @@ defmodule SnowflexTest do
 
   test "literals" do
     query = "schema" |> where(foo: true) |> select([], true) |> plan()
-    assert all(query) == ~s{SELECT TRUE FROM `schema` AS s0 WHERE (s0.`foo` = TRUE)}
+    assert all(query) == ~s{SELECT TRUE FROM schema AS s0 WHERE (s0.foo = TRUE)}
 
     query = "schema" |> where(foo: false) |> select([], true) |> plan()
-    assert all(query) == ~s{SELECT TRUE FROM `schema` AS s0 WHERE (s0.`foo` = FALSE)}
+    assert all(query) == ~s{SELECT TRUE FROM schema AS s0 WHERE (s0.foo = FALSE)}
 
     query = "schema" |> where(foo: "abc") |> select([], true) |> plan()
-    assert all(query) == ~s{SELECT TRUE FROM `schema` AS s0 WHERE (s0.`foo` = 'abc')}
+    assert all(query) == ~s{SELECT TRUE FROM schema AS s0 WHERE (s0.foo = 'abc')}
 
     query = "schema" |> where(foo: 123) |> select([], true) |> plan()
-    assert all(query) == ~s{SELECT TRUE FROM `schema` AS s0 WHERE (s0.`foo` = 123)}
+    assert all(query) == ~s{SELECT TRUE FROM schema AS s0 WHERE (s0.foo = 123)}
 
     query = "schema" |> where(foo: 123.0) |> select([], true) |> plan()
-    assert all(query) == ~s{SELECT TRUE FROM `schema` AS s0 WHERE (s0.`foo` = (0 + 123.0))}
+    assert all(query) == ~s{SELECT TRUE FROM schema AS s0 WHERE (s0.foo = (0 + 123.0))}
   end
 
   test "tagged type" do
     query =
-      Schema |> select([], type(^"601d74e4-a8d3-4b6e-8365-eddb4c893327", Ecto.UUID)) |> plan()
+      TestSchema |> select([], type(^"601d74e4-a8d3-4b6e-8365-eddb4c893327", Ecto.UUID)) |> plan()
 
-    assert all(query) == ~s{SELECT CAST(? AS binary(16)) FROM `schema` AS s0}
+    assert all(query) == ~s{SELECT CAST(? AS binary(16)) FROM schema AS s0}
```

Comment:
We have a small issue here as Snowflake only support binaries as hex-encoded strings, so this should technically be varchar. I will update the `Ecto_to_db` functions in `connection.ex` to correct this today, then I think you will need to update on top of that.

Reply:
My branch has been updated.